examples/energy-management-app/energy-management-common/energy-management-app.zap

```diff
@@ -3470,7 +3470,7 @@
               "storageOption": "External",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "",
+              "defaultValue": null,
```

Comment:
In the Spec, SupportedModes does not have the nullable quality `X`, so shouldn't thisbe left empty?